Specifications/QIR/Data-Types.md

```diff
@@ -96,8 +94,7 @@ simple types:
 | Function                          | Signature                | Description |
 |-----------------------------------|--------------------------|-------------|
 | __quantum__rt__result_equal       | `i1(%Result*, %Result*)` | Returns true if the two results are the same, and false if they are different. |
-| __quantum__rt__result_reference   | `void(%Result*)`         | Increments the reference count of a Result pointer. |
-| __quantum__rt__result_unreference | `void(%Result*)`         | Decrements the reference count of a Result pointer and releases the result if appropriate. |
+| __quantum__rt__result_update_reference_count   | `void(%Result*, i64)` | Adds the given integer value to the reference count for the result. Deallocates the result if the reference count becomes 0. The behavior is undefined if the reference count becomes negative. |
```

Comment:
I've recently standardized the runtime to use i32 for ref/alias counts. Any reason we should be supporting i64?

Reply:
No, no reason besides that we are using i64 by default throughout QIR.

Reply:
I think you are right, i32 is enough and we are already using it in other places. I updated to i32.